themes/generic/src/main/assets/js/warnOldVersion.js

```diff
@@ -0,0 +1,22 @@
+function initOldVersionWarnings($, thisVersion, projectUrl) {
+    var schemeLessUrl = projectUrl;
+    if (projectUrl.startsWith("http://")) projectUrl = schemeLessUrl.substring(5);
+    else if (projectUrl.startsWith("https://")) projectUrl = schemeLessUrl.substring(6);
+    const url = schemeLessUrl + "/paradox.json";
+    console.log("trying to read version data from " + url);
```

Comment:
I think this can go. When debuging it is possible to see if and where the request is sent in the `Network` tab.